### PR TITLE
feat(atomic): add new prop to allow to control the iframe sandbox attribute

### PR DESCRIPTION
**Pull request by olamothe**
https://coveord.atlassian.net/browse/KIT-2203

### CODE REVIEW

File: packages/atomic/src/components/search/result-template-components/atomic-quickview/atomic-quickview.tsx [hunk 1]
```diff
@@ -37,6 +37,20 @@ export class AtomicQuickview implements InitializableComponent {
   @State()
   public quickviewState!: QuickviewState;
 
+  /**
```
Comment (olamothe) [inline packages/atomic/src/components/search/result-template-components/atomic-quickview/atomic-quickview.tsx:40]: @jpmarceau 🙇 Thanks !

**Review by louis-bompart — COMMENTED** [review]
> `allow-same-origin` is not optional, and must always be included in the list of allowed capabilities in order for the component to function properly.

Should we dummy proof?
  a. Add it if the user forgot it
  b. Throw a warning/error to the user
  c. All the above
  d. Obi-Wan Kenobi

**Review by ThibodeauJF — APPROVED** [review]

File: packages/atomic/src/components/search/result-template-components/atomic-quickview/atomic-quickview.tsx [hunk 2]
```diff
@@ -37,6 +37,20 @@ export class AtomicQuickview implements InitializableComponent {
...
+   * The sandbox attribute to apply to the quickview iframe.
+   *
+   * The quickview is loaded inside an iframe with a [sandbox](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#attr-sandbox) attribute for security reasons.
+   *
+   * This attribute primarily exists in order to protect against potential XSS attacks that could originate from the document being displayed.
+   *
+   * By default, the sandbox attributes are: `allow-popups allow-top-navigation allow-same-origin`.
+   *
+   * `allow-same-origin` is not optional, and must always be included in the list of allowed capabilities in order for the component to function properly.
```
Comment (jpmarceau) [inline packages/atomic/src/components/search/result-template-components/atomic-quickview/atomic-quickview.tsx:49]: Clear and well-written description Oli 👌 
I only have minor suggestions:

```suggestion
   * The `sandbox` attribute to apply to the quickview iframe.
   *
   * The quickview is loaded inside an iframe with a [`sandbox`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#attr-sandbox) attribute for security reasons.
   *
   * This attribute exists primarily to protect against potential XSS attacks that could originate from the document being displayed.
   *
   * By default, the sandbox attributes are: `allow-popups allow-top-navigation allow-same-origin`.
   *
   * `allow-same-origin` is not optional, and must always be included in the list of allowed capabilities for the component to function properly.
```

**Review by jpmarceau — APPROVED** [review]

File: packages/atomic/src/components/search/result-template-components/atomic-quickview/atomic-quickview.tsx [hunk 3]
```diff
@@ -37,6 +37,20 @@ export class AtomicQuickview implements InitializableComponent {
...
+  /**
+   * The `sandbox` attribute to apply to the quickview iframe.
+   *
+   * The quickview is loaded inside an iframe with a [`sandbox`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#attr-sandbox) attribute for security reasons.
+   *
+   * This attribute exists primarily to protect against potential XSS attacks that could originate from the document being displayed.
+   *
+   * By default, the sandbox attributes are: `allow-popups allow-top-navigation allow-same-origin`.
+   *
+   * `allow-same-origin` is not optional, and must always be included in the list of allowed capabilities for the component to function properly.
```
Comment (btaillon) [inline packages/atomic/src/components/search/result-template-components/atomic-quickview/atomic-quickview.tsx:49]: I think we should also add validation for that.

**Review by btaillon — APPROVED** [review]

**Pull request merged**